docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -1,246 +1,211 @@
 .. _real_and_lazy_data:
 
+
+.. testsetup:: *
+
+    import dask.array as da
+    import iris
+    import numpy as np
+
+
 ==================
 Real and Lazy Data
 ==================
 
-What is Real and Lazy Data?
----------------------------
+We have seen in the :doc:`user_guide_introduction` section of the user guide that
+Iris cubes contain data and metadata about a phenomenon. The data attribute of a
+cube contains the actual numerical values recorded for the phenomenon the cube describes.
```

Comment:
I tried saying this sentence out loud (which I can do because I am at home...) and my tongue fell over itself, just like my brain did when I said it in my head.  I very much agree with this introduction, but I think it needs less words in it.